components/candle/challengeHelpers.ts

```diff
@@ -111,13 +111,13 @@ export function inclusionDataCheck(
     if (!incData) return true
     if (!contract) return false
 
-    return (
+    return Boolean(
```

Comment:
Why it couldn't infer this is beyond me. It's pretty obvious.

Reply:
It thought the last item in the chain was returning the value, which it technically was, but that still could be cast to a boolean in conditions